Automattic/crowdsignal-forms
# Use Notice component for SignalWarning

This PR:

  - introduces EditorNotice as a wrapper around Notice (`@wordpress/components/notice`)
  - turns SignalWarning into a of wrapper around EditorNotice for consistency
  - tweak error notice button style and text: now it reads "Retry"

It also turns some styles used inside the Notice into more generic styles so all our notices have a consisten spacing:
![image](https://user-images.githubusercontent.com/157240/107801722-3422df80-6d3f-11eb-900e-fe6112372362.png)


## Test instructions
Checkout and `make client`. Use a free account with exhausted signals to trigger the warning. See that it looks good and inline with any other notice present.
